medusa/app.py

```diff
@@ -3,642 +3,657 @@
+            'udp://tracker.internetwarriors.net:1337/announce', 'udp://ipv4.tracker.harry.lu:80/announce',
+            'http://ipv4.tracker.harry.lu:80/announce', 'udp://mgtracker.org:2710/announce',
+            'http://mgtracker.org:6969/announce', 'udp://tracker.mg64.net:6969/announce',
+            'http://tracker.mg64.net:6881/announce', 'http://torrentsmd.com:8080/announce'
+        ]
+
+        self.REQUIRE_WORDS = []
+        self.IGNORED_SUBS_LIST = ['dk', 'fin', 'heb', 'kor', 'nor', 'nordic', 'pl', 'swe']
+        self.IGNORE_UND_SUBS = False
+        self.SYNC_FILES = ['!sync', 'lftp-pget-status', 'part', 'bts', '!qb', '!qB']
+
+        self.CALENDAR_UNPROTECTED = False
+        self.CALENDAR_ICONS = False
+        self.NO_RESTART = False
+
+        self.TMDB_API_KEY = 'edc5f123313769de83a71e157758030b'
+        # TRAKT_API_KEY = 'd4161a7a106424551add171e5470112e4afdaf2438e6ef2fe0548edc75924868'
+
+        self.TRAKT_API_KEY = '5c65f55e11d48c35385d9e8670615763a605fad28374c8ae553a7b7a50651ddd'
+        self.TRAKT_API_SECRET = 'b53e32045ac122a445ef163e6d859403301ffe9b17fb8321d428531b69022a82'
+        self.TRAKT_PIN_URL = 'https://trakt.tv/pin/4562'
+        self.TRAKT_OAUTH_URL = 'https://trakt.tv/'
+        self.TRAKT_API_URL = 'https://api.trakt.tv/'
+
+        self.FANART_API_KEY = '9b3afaf26f6241bdb57d6cc6bd798da7'
+
+        self.SHOWS_RECENT = []
+
+        self.__INITIALIZED__ = False
+
+        self.NEWZNAB_PROVIDERS = []
+
+        self.TORRENTRSS_PROVIDERS = []
+
+        self.TORZNAB_PROVIDERS = []
+
+        self.RECENTLY_DELETED = set()
+
+        self.RECENTLY_POSTPROCESSED = {}
+
+        self.RELEASES_IN_PP = []
+
+        self.PRIVACY_LEVEL = 'normal'
 
-PROPERS_INTERVAL_LABELS = {'daily': '24 hours',
-                           '4h': '4 hours',
-                           '90m': '90 mins',
-                           '45m': '45 mins',
-                           '15m': '15 mins'
-                           }
+        self.PROPERS_SEARCH_INTERVAL = {'15m': 15, '45m': 45, '90m': 90, '4h': 4 * 60, 'daily': 24 * 60}
 
-# Plex fallback settings
-FALLBACK_PLEX_ENABLE = True
-FALLBACK_PLEX_NOTIFICATIONS = True
-FALLBACK_PLEX_TIMEOUT = 3
-FALLBACK_PLEX_API_URL = 'https://tvdb2.plex.tv'
-TVDB_API_KEY = '0629B785CE550C8D'
+        self.PROPERS_INTERVAL_LABELS = {
+            'daily': '24 hours',
+            '4h': '4 hours',
+            '90m': '90 mins',
+            '45m': '45 mins',
+            '15m': '15 mins'
+        }
+
+        # Plex fallback settings
+        self.FALLBACK_PLEX_ENABLE = True
+        self.FALLBACK_PLEX_NOTIFICATIONS = True
+        self.FALLBACK_PLEX_TIMEOUT = 3
+        self.FALLBACK_PLEX_API_URL = 'https://tvdb2.plex.tv'
+        self.TVDB_API_KEY = '0629B785CE550C8D'
+
+
+app = MedusaApp()
+for app_key, app_value in viewitems(app.__dict__):
+    setattr(sys.modules[__name__], app_key, app_value)
```

Comment:
This feels a little hacky, I agree. But there aren't allot of alternatives, then to change all imports in the app.
This puts all the classes properties on the module.

Whe'll have to test how this works with getters/setters.